myplanet/skeletor
# 8.2.x: Media update

Main goal of the story is to transition from using the entity browser to the core media library. There are also some minor fixes to media object component.

### Todo
- [x] Remove entity browser 
- [x] Use core media library
- [x] Test with Drupal 8.8.x
- [x] Fix focal point 
- [ ] Test updating existing site that uses entity browser

### Focal Point
When it comes to fixing focal point, I could not get it to work when you add media during the node creation or editing process:
<img width="1189" alt="Capture d'écran 2019-12-24 10 40 48" src="https://user-images.githubusercontent.com/585729/71417087-df536f80-2639-11ea-97bf-d793feaa4b56.png">


However it does work when editing the media entity directly (`/admin/content/media`):
<img width="1319" alt="Capture d'écran 2019-12-24 10 39 58" src="https://user-images.githubusercontent.com/585729/71417086-df536f80-2639-11ea-97f4-db3f4597e1b2.png">

This is probably related to this focal point issue: https://www.drupal.org/project/focal_point/issues/3094478

### Testing Focal Point

- Create new content type
- Add a paragraph field for the slideshow component
- Create a node for your new content type
- Add media
- When image is uploaded there should be a crosshair over the image
